### PR TITLE
Added option to toggle "enchantability" of an item

### PR DESCRIPTION
**Pull request by Poslovitch**
Related to https://github.com/TheBusyBiscuit/Slimefun4/issues/280

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Doesn't this still need to handle this boolean in the machine?

**Comment by Poslovitch** [comment]
I don't know. What is the problem exactly?

**Comment by TheBusyBiscuit** [comment]
Well, you added a boolean.

But that boolean is not used anywhere...

**Comment by Poslovitch** [comment]
Well, it seems I forgot to push the Enchanter/disenchanter files with the SlimefunItem x)
I push them in about 10 minutes

File: src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/AutoEnchanter.java [hunk 1]
```diff
@@ -92,6 +93,7 @@ protected void tick(Block b) {
 			slots:
 			for (int slot: getInputSlots()) {
 				ItemStack target = BlockStorage.getInventory(b).getItemInSlot(slot == getInputSlots()[0] ? getInputSlots()[1]: getInputSlots()[0]);
+				if(SlimefunItem.getByItem(target) != null && !SlimefunItem.getByItem(target).isEnchantable()) return;
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/AutoEnchanter.java]: Heavy call right there.

Should be split into a variable for performance reasons.

**Pull request merged**